### ISSUE TITLE
Clojure 1.10 failing

### ISSUE THREAD

**Issue by Futurile** [bfontaine/lein-fore-prob#7]
I'm trying to get lein-fore-prob running using Clojure 1.10 on Linux. When I try to use it I get this:

```
lein fore-prob 43
clojure.lang.Compiler$CompilerException: Syntax error macroexpanding clojure.core/defn- at (clj_tuple.clj:340:1).
#:clojure.error{:phase :macro-syntax-check, :line 340, :column 1, :source "clj_tuple.clj", :symbol clojure.core/defn-}
```
Then a lot of output and later this:
```
Caused by: clojure.lang.ExceptionInfo: Call to clojure.core/defn- did not conform to spec.
#:clojure.spec.alpha{:problems [{:path [:fn-name], :pred clojure.core/simple-symbol?, :val clj-tuple/conj-tuple, :via [:clojure.core.specs.alpha/defn-args :clojure.core.specs.alpha/defn-args], :in [0]}], :spec #object[clojure.spec.alpha$regex_spec_impl$reify__2509 0x40d60f2 "clojure.spec.alpha$regex_spec_impl$reify__2509@40d60f2"], :value (clj-tuple/conj-tuple [t__1976__auto__ x__1977__auto__] (clojure.core/let [t__1976__auto__ t__1976__auto__] (clojure.core/case (.count t__1976__auto__) 0 (new Tuple1 x__1977__auto__ nil) 1 (new Tuple2 (. t__1976__auto__ e0) x__1977__auto__ nil) 2 (new Tuple3 (. t__1976__auto__ e0) (. t__1976__auto__ e1) x__1977__auto__ nil) 3 (new Tuple4 (. t__1976__auto__ e0) (. t__1976__auto__ e1) (. t__1976__auto__ e2) x__1977__auto__ nil) 4 (new Tuple5 (. t__1976__auto__ e0) (. t__1976__auto__ e1) (. t__1976__auto__ e2) (. t__1976__auto__ e3) x__1977__auto__ nil) 5 (new Tuple6 (. t__1976__auto__ e0) (. t__1976__auto__ e1) (. t__1976__auto__ e2) (. t__1976__auto__ e3) (. t__1976__auto__ e4) x__1977__auto__ nil) 6 (clojure.core/conj (clojure.lang.PersistentVector/create t__1976__auto__) x__1977__auto__)))), :args (clj-tuple/conj-tuple [t__1976__auto__ x__1977__auto__] (clojure.core/let [t__1976__auto__ t__1976__auto__] (clojure.core/case (.count t__1976__auto__) 0 (new Tuple1 x__1977__auto__ nil) 1 (new Tuple2 (. t__1976__auto__ e0) x__1977__auto__ nil) 2 (new Tuple3 (. t__1976__auto__ e0) (. t__1976__auto__ e1) x__1977__auto__ nil) 3 (new Tuple4 (. t__1976__auto__ e0) (. t__1976__auto__ e1) (. t__1976__auto__ e2) x__1977__auto__ nil) 4 (new Tuple5 (. t__1976__auto__ e0) (. t__1976__auto__ e1) (. t__1976__auto__ e2) (. t__1976__auto__ e3) x__1977__auto__ nil) 5 (new Tuple6 (. t__1976__auto__ e0) (. t__1976__auto__ e1) (. t__1976__auto__ e2) (. t__1976__auto__ e3) (. t__1976__auto__ e4) x__1977__auto__ nil) 6 (clojure.core/conj (clojure.lang.PersistentVector/create t__1976__auto__) x__1977__auto__))))}
```
I'm using the latest Lein and OpenJDK:
```
lein version
Leiningen 2.9.1 on Java 11.0.2 OpenJDK 64-Bit Server VM
```
The dependencies are:
```
lein deps :tree
[[org.clojure/clojure]]]
 [nrepl "0.6.0" :exclusions [[org.clojure/clojure]]]
 [org.clojure/clojure "1.10.0"]
   [org.clojure/core.specs.alpha "0.2.44"]
   [org.clojure/spec.alpha "0.2.176"]
```
You committed 0.1.3 to the repo last year, but I'm having to specify:

```
{:user {:plugins [ ; ... other plugins ...
                  [lein-fore-prob "0.1.2"]]}}
```

Otherwise it says it can't find it on Clojars.

**Comment by bfontaine** [comment]
Thank you for the report! Looks like `clj-tuple` is breaking everything: https://github.com/ztellman/clj-tuple/issues/18.

I bumped `clj-http` to the latest version, which use the latest `clj-tuple`; it should fix the issue.

I just released `0.1.3` (for real this time): https://clojars.org/lein-fore-prob

**Comment by Futurile** [comment]
Yes that work, thanks so much - appreciate it.
